shap/plots/_force.py

```diff
@@ -257,16 +259,18 @@ def save_html(out_file, plot, full_html=True):
     Parameters
     ----------
     out_file : str or file
-        Location or file to be written to
+        Location or file to be written to.
+
     plot : BaseVisualizer
-        Visualizer returned by shap.force_plot()
+        Visualizer returned by :func:`shap.plots.force()`.
+
     full_html : boolean (default: True)
-        If True, writes a complete HTML document starting
-        with an <html> tag. If False, only script and div
+        If ``True``, writes a complete HTML document starting
+        with an ``<html>`` tag. If ``False``, only script and div
         tags are included.
     """
 
-    assert isinstance(plot, BaseVisualizer), "save_html requires a Visualizer returned by shap.force_plot()."
+    assert isinstance(plot, BaseVisualizer), "`save_html` requires a Visualizer returned by `shap.plots.force()`."
```

Comment:
Nitpick: do backticks have any affect here? As the message would appear in an Exception traceback, presumably there wouldn't be any formatting applied.

Reply:
Nope, no effect. It's just to indicate to the user that it is a function object and not "text".

Guess it doesnt really matter here, since it's unlikely that the user would mistake this as an English word. In other cases (say our `sample` function), it would be confusing to read if we didnt add backticks.